Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -120,22 +122,19 @@ public function interpret(array $data, $associative = true)
                 $entities[$index] = $entity;
                 $this->internalLookupCache[] = $entity;
             } catch (InterpreterException $exception) {
-                $this->handleInterpreterError($exception, $index, $row);
+                $exceptionEvent = new ExceptionEvent($this->log, $exception, $index);
```

Comment:
Ok. Makes sense. But then I guess I would have to do the same on the other places that I create events. Like the csvImporter, which already has 7 dependencies. So If I add 2 more -because in that class I create both `exceptionEvent` and `errorEvent` we will reach 9, which I guess is not that good. 

I prefer not to create factories in this PR because it's just an intermediate step.